CDCgov/prime-reportstream
# 8587 Added additional CA receiver transforms

This PR adds additional CA receiver transforms to be able to send ORC to CA

Test Steps:
1. Check out and build this branch
2. Run `./gradlew testIntegration`
3. Submit the generated sample  HL7 v2 message from Simple Report found under `/src/testIntegration/resources/datatests/FHIR_to_HL7/sample_SR_1_20230302-0001.hl7` to the [CalREDIE validation tool](https://calrediestaging.cdph.ca.gov/webvalidator/) and verify that no errors are returned.

## Changes
- Added additional CA receiver transforms

## Checklist

### Testing

### Process

## Linked Issues
- #8587 

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #issue

## Specific Security-related subjects a reviewer should pay specific attention to
- Does this PR introduce new endpoints?
    - new endpoint A
    - new endpoint B
- Does this PR include changes in authentication and/or authorization of existing endpoints?
- Does this change introduce new dependencies that need vetting?
- Does this change require changes to our infrastructure?
- Does logging contain sensitive data?
- Does this PR include or remove any sensitive information itself?

If you answered '_yes_' to any of the questions above, conduct a detailed Review that addresses at least:

- What are the potential security threats and mitigations? Please list the _STRIDE_ threats and how they are mitigated
    - **S**poofing (faking authenticity)
        - Threat _T_, which could be achieved by _A_, is mitigated by _M_
    - **T**ampering (influence or sabotage the integrity of information, data, or system)
    - **R**epudiation (the ability to dispute the origin or originator of an action)
    - **I**nformation disclosure (data made available to entities who should not have it)
    - **D**enial of service (make a resource unavailable)
    - **E**levation of Privilege (reduce restrictions that apply or gain privileges one should not have)
- Have you ensured logging does not contain sensitive data?
- Have you received any additional approvals needed for this change?
